src/components/modal/Modal.scss

```diff
@@ -151,7 +151,9 @@
         }
 
         .modal-close-button {
+            top: initial;
```

Comment:
minor but `initial` isn't supported in IE11. Although I'm not sure if we still care to support it since this is for responsive design and IE11 is near EOL.

Reply:
I think `top: auto` would be a good replacement as that should be the initial value in this case.

Reply:
Done